pkg/cli/config/config.go

```diff
@@ -154,17 +165,26 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 		bpConfig.ProxyURL = &proxyURL
 	}
 
+	if (bpConfig.Govcloud) {
+		str := ""
+		bpConfig.ProxyURL = &str
+	}
+
 	bpConfig.SessionDirectory = viper.GetString("session-dir")
 	bpConfig.AssumeInitialArn = viper.GetString("assume-initial-arn")
 	bpConfig.DisplayClusterInfo = viper.GetBool("display-cluster-info")
 	bpConfig.DisableKubePS1Warning = viper.GetBool("disable-kube-ps1-warning")
 
-	// pagerDuty token is optional
-	pagerDutyAPIKey := viper.GetString("pd-key")
-	if pagerDutyAPIKey != "" {
-		bpConfig.PagerDutyAPIKey = pagerDutyAPIKey
+	// pagerDuty token is optional. Don't even check for FedRAMP
+	if !(bpConfig.Govcloud) {
```

Comment:
As I understand, most configs may not be related to govCloud clusters ,  if we can switch the BackplaneConfiguration struct based on `bpConfig.Govcloud` which may solve most of the issues. ( Just a thought ) 

my thinking is to switch the struct based on the type  

```
BackplaneManagedClusterConfiguration{
 proxy-url
 pd-key
 ...
 }
 
 BackplaneGovCloudConfiguration{
 gov-cloud-configs
 .....
 }

Reply:
Yeah, this sounds a better solution